R/setPrior.R

```diff
@@ -0,0 +1,105 @@
+
+#priors() method for stanmodule, hazardlink , longmodel and osmodel
+
+# getter method for LongModel:
+setGeneric(name = "prior", def = function(object) standardGeneric("prior"))
```

Comment:
please add generic definitions to `generics.R` instead of here

Reply:
also please use plural form `priors` for all generics and methods

Reply:
Sure! Also move the generic definitions of setGeneric(name = "priors<-",...) to generics.R, right?

Reply:
yep exactly :-)